### DISCUSSION TITLE
原始订单数据从MySQL同步到Doris，订单状态更新或者订单删除

### DISCUSSION THREAD

**Discussion by jacentsao** [apache/doris#11270] [A - General / Q&A]
数据流向为MySQL -> Flink CDC -> Doris ODS层，使用UNIQUE模型保证数据与原始数据一致。

请问一下推荐怎么从ODS层生成按天汇总的数据了，UNIQUE模型不支持聚合，大家都是定时任务跑的吗，
还是说在Flink那边通过流处理先计算好再入库更新汇总的数据了？

有没有实时的方案，搜了下只搜到一个  [Doris + Flink + DophinScheduler + Dinky 构建开源数据平台](https://www.bilibili.com/video/BV1JG4y1i7Bt?spm_id_from=333.999.0.0)，官方推荐的标准方案是什么了。

**Comment by catpineapple** [comment]
can i have your contact information,like WeChat? We will make the design after we understand the specific details.
